### DISCUSSION TITLE
how can I create a multi-window application

### DISCUSSION THREAD

**Discussion by shi-yan** [gfx-rs/wgpu#2388] [Q&A]
I can't find information regarding how to create multi-window application,

for opengl, for example, usually we create multiple windows using a shared context.

but there doesn't seem to be shared context with wgpu.

And I read that for wgpu, there is no swapchain. We should use surface instead. 

is there an example? how should I do it?

**Comment by cwfitzgerald** [comment]
The hello-windows example (https://github.com/gfx-rs/wgpu/tree/master/wgpu/examples/hello-windows) should have what you need.
